analysis.js

```diff
@@ -8,22 +8,21 @@
  ** Environment Variables
  ** In order to use this analysis, you must setup the Environment Variable table.
  **
- ** account_token: Your account token
  ** checkin_time: Minutes between the last input of the device before sending the notification.
  ** tag_key: Device tag Key to filter the devices.
  ** tag_value: Device tag Value to filter the devices.
  ** email_list: Email list comma separated.
  ** sms_list: Phone number list comma separated. The phone number must include the country code
  **
- ** Steps to generate an account_token:
- ** 1 - Enter the following link: https://admin.tago.io/account/
- ** 2 - Select your Profile.
- ** 3 - Enter Tokens tab.
- ** 4 - Generate a new Token with Expires Never.
- ** 5 - Press the Copy Button and place at the Environment Variables tab of this analysis.
+ ** How to use:
+ ** To analysis works, you need to add a new policy in your account. Steps to add a new policy:
+ **  1 - Click the button "Add Policy" at this url: https://admin.tago.io/am;
+ **  2 - In the Target selector, select the Analysis with the field set as "ID" and choose your Analysis in the list;
```

Comment:
** 2 - In the Target selector, with the field set as "ID", choose your Analysis in the list;